### PR TITLE
Doc refactoring

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
## Summary of the Pull Request
We need this for crossplane integration

## PR Checklist

* [ ] Refers to: #xxx
* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.
* [x] Release notes added.

### CODE REVIEW

File: docs/data-sources/cce_cluster_kubeconfig_v3.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Cloud Container Engine (CCE)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_cce_cluster_kubeconfig_v3"
+sidebar_current: "docs-opentelekomcloud-datasource-cce-cluster-kubeconfig-v3"
+description: |-
+Get CCE a cluster's kubeconfig file from OpenTelekomCloud
```
Comment (artem-lifshits) [inline docs/data-sources/cce_cluster_kubeconfig_v3.md:7]: missing doc link

File: docs/data-sources/dcs_certificate_v2.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ---
 subcategory: "Distributed Cache Service (DCS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_dcs_certificate_v2"
+sidebar_current: "docs-opentelekomcloud-datasource-dcs-certificate-v1"
```
Comment (artem-lifshits) [inline docs/data-sources/dcs_certificate_v2.md]: api version diff

**Review by sgmv — DISMISSED** [review]

File: docs/data-sources/dws_flavors_v2.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Data Warehouse Service (DWS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_dws_flavors_v2"
+sidebar_current: "docs-opentelekomcloud-datasource-dms-flavors-v2"
+description: |-
+Get details about DWS flavors from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/dws_flavors_v2.md:8]: missing link to api doc

File: docs/data-sources/identity_auth_scope_v3.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Identity and Access Management (IAM)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_identity_auth_scope_v3"
+sidebar_current: "docs-opentelekomcloud-datasource-identity-auth-scope-v3"
+description: |-
+Get authentication information about the current auth scope in use within OpenTelekomCloud
```
Comment (artem-lifshits) [inline docs/data-sources/identity_auth_scope_v3.md:7]: missing link to api doc

File: docs/data-sources/identity_role_custom_v3.md [hunk 5]
```diff
@@ -1,7 +1,15 @@
 ---
 subcategory: "Identity and Access Management (IAM)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_identity_role_custom_v3"
+sidebar_current: "docs-opentelekomcloud-datasource-identity-role-custom-v3"
+description: |-
+Get a IAM role information from OpenTelekomCloud
```
Comment (artem-lifshits) [inline docs/data-sources/identity_role_custom_v3.md]: custom IAM role (opposed to predefined system one in `data_source/opentelekomcloud_identity_role_v3`

File: docs/data-sources/kms_data_key_v1.md [hunk 6]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Key Management Service (KMS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_kms_data_key_v1"
+sidebar_current: "docs-opentelekomcloud-datasource-kms-data-key-v1"
+description: |-
+Get an available KMS DEK from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/kms_data_key_v1.md:8]: missing link to api doc

File: docs/data-sources/rts_software_config.md [hunk 7]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Resource Template Service (RTS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rts_software_config_v1"
+sidebar_current: "docs-opentelekomcloud-datasource-rts-software-config-v1"
+description: |-
+Get details about a specific RTS Software Config from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/rts_software_config.md:9]: link to api doc missing

File: docs/data-sources/rts_software_deployment.md [hunk 8]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Resource Template Service (RTS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rts_software_deployment_v1"
+sidebar_current: "docs-opentelekomcloud-datasource-rts-software-deployment-v1"
+description: |-
+Get details about a specific RTS Software Deployment from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/rts_software_deployment.md:9]: same

File: docs/data-sources/rts_stack_resource_v1.md [hunk 9]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Resource Template Service (RTS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rts_stack_resource_v1"
+sidebar_current: "docs-opentelekomcloud-datasource-rts-stack-resource-v1"
+description: |-
+Get details about a specific RTS resource metadata from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/rts_stack_resource_v1.md:9]: same

File: docs/data-sources/rts_stack_v1.md [hunk 10]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Resource Template Service (RTS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_rts_stack_v1"
+sidebar_current: "docs-opentelekomcloud-datasource-rts-stack-v1"
+description: |-
+Get details about a specific RTS resource from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/rts_stack_v1.md:9]: same

File: docs/data-sources/s3_bucket_object.md [hunk 11]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Object Storage Service (S3)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_s3_bucket_object"
+sidebar_current: "docs-opentelekomcloud-datasource-s3-bucket-object"
+description: |-
+Get details about S3 object metadata and content from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/s3_bucket_object.md:9]: link to api doc missing

File: docs/data-sources/vpc_bandwidth_v2.md [hunk 12]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_vpc_bandwidth_v2"
+sidebar_current: "docs-opentelekomcloud-datasource-vpc-bandwidth-v2"
+description: |-
+Get details about a specific shared bandwidth from OpenTelekomCloud
 ---
```
Comment (artem-lifshits) [inline docs/data-sources/vpc_bandwidth_v2.md:9]: link to api doc missing

File: docs/resources/blockstorage_volume_attach_v2.md [hunk 13]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Elastic Volume Service (EVS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_blockstorage_volume_attach_v2"
+sidebar_current: "docs-opentelekomcloud-resource-blockstorage-volume-attach-v2"
+description: |-
+Manages a BlockStorage volume attach resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/blockstorage_volume_attach_v2.md:9]: missing link to api doc

File: docs/resources/blockstorage_volume_v2.md [hunk 14]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Elastic Volume Service (EVS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_blockstorage_volume_v2"
+sidebar_current: "docs-opentelekomcloud-resource-blockstorage-volume-v2"
+description: |-
+Manages a BlockStorage volume resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/blockstorage_volume_v2.md:9]: missing link to api doc

File: docs/resources/compute_bms_server_v2.md [hunk 15]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Bare Metal Server (BMS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_compute_bms_server_v2"
+sidebar_current: "docs-opentelekomcloud-resource-compute-bms-server-v2"
+description: |-
+Manages a BMS Server resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/compute_bms_server_v2.md:9]: missing link to api doc

File: docs/resources/compute_bms_tags_v2.md [hunk 16]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Bare Metal Server (BMS)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_compute_bms_tags_v2"
+sidebar_current: "docs-opentelekomcloud-resource-compute-bms-tags-v2"
+description: |-
+Manages a BMS Tags resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/compute_bms_tags_v2.md:9]: same

File: docs/resources/fw_policy_v2.md [hunk 17]
```diff
@@ -1,9 +1,14 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_fw_policy_v2"
+sidebar_current: "docs-opentelekomcloud-resource-fw-firewall-policy-v2"
```
Comment (artem-lifshits) [inline docs/resources/fw_policy_v2.md]: docs-opentelekomcloud-resource-fw-**firewall**-policy-v2

File: docs/resources/fw_rule_v2.md [hunk 18]
```diff
@@ -1,9 +1,14 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_fw_rule_v2"
+sidebar_current: "docs-opentelekomcloud-resource-fw-firewall-rule-v2"
```
Comment (artem-lifshits) [inline docs/resources/fw_rule_v2.md]: docs-opentelekomcloud-resource-fw-**firewall**-rule-v2

File: docs/resources/nat_snat_rule_v2.md [hunk 19]
```diff
@@ -1,9 +1,13 @@
----
 subcategory: "NAT"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_nat_snat_rule_v2"
+sidebar_current: "docs-opentelekomcloud-resource-nat-snat-rule-v2"
+description: |-
+Manages a NAT SNAT Rule resource within OpenTelekomCloud.
 ---
 
 Up-to-date reference of API arguments for NAT snat you can get at
```
Comment (artem-lifshits) [inline docs/resources/nat_snat_rule_v2.md]: NAT SNAT (uppercase)

File: docs/resources/networking_port_secgroup_associate_v2.md [hunk 20]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_networking_port_secgroup_associate_v2"
+sidebar_current: "docs-opentelekomcloud-resource-networking-port-secgroup-associate-v2"
+description: |-
+Manages a VPC Port's Security Groups resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/networking_port_secgroup_associate_v2.md:9]: missing link o api doc

File: docs/resources/networking_vip_associate_v2.md [hunk 21]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_networking_vip_associate_v2"
+sidebar_current: "docs-opentelekomcloud-resource-networking-vip-associate-v2"
+description: |-
+Manages a VPC VIP Associate resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/networking_vip_associate_v2.md:9]: missing link to api doc

File: docs/resources/networking_vip_v2.md [hunk 22]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Virtual Private Cloud (VPC)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_networking_vip_v2"
+sidebar_current: "docs-opentelekomcloud-resource-networking-vip-v2"
+description: |-
+Manages a VPC VIP resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/networking_vip_v2.md:9]: missing link to api doc

File: docs/resources/s3_bucket.md [hunk 23]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Object Storage Service (S3)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_s3_bucket"
+sidebar_current: "docs-opentelekomcloud-resource-s3-bucket"
+description: |-
+Manages an S3 Bucket resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/s3_bucket.md:9]: missing link to api doc

File: docs/resources/s3_bucket_object.md [hunk 24]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Object Storage Service (S3)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_s3_bucket_object"
+sidebar_current: "docs-opentelekomcloud-resource-s3-bucket-object"
+description: |-
+Manages an S3 Bucket Object resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/s3_bucket_object.md:9]: missing link to api doc

File: docs/resources/s3_bucket_policy.md [hunk 25]
```diff
@@ -1,5 +1,10 @@
 ---
 subcategory: "Object Storage Service (S3)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_s3_bucket_policy"
+sidebar_current: "docs-opentelekomcloud-resource-s3-bucket-policy"
+description: |-
+Manages an S3 Bucket Policy resource within OpenTelekomCloud.
 ---
```
Comment (artem-lifshits) [inline docs/resources/s3_bucket_policy.md:9]: missing link to api doc

File: docs/resources/waf_dedicated_alarm_masking_rule_v1.md [hunk 26]
```diff
@@ -1,9 +1,14 @@
...
+sidebar_current: "docs-opentelekomcloud-resource-waf-dedicated-alarm-masking-rule-v1"
+description: |-
+Manages a WAF Dedicated False Alarm Masking Rule resource within OpenTelekomCloud.
 ---
 
 Up-to-date reference of API arguments for WAF dedicated Global Protection Whitelist (formerly False Alarm Masking) rule you can get at
-[Official Docs Portal](https://docs.otc.t-systems.com/web-application-firewall-dedicated/api-ref/apis/rule_management/creating_an_information_leakage_protection_rule.html).
+[documentation portal](https://docs.otc.t-systems.com/web-application-firewall-dedicated/api-ref/apis/rule_management/creating_an_information_leakage_protection_rule.html).
 
 # opentelekomcloud_waf_dedicated_anti_leakage_rule_v1
```
Comment (artem-lifshits) [inline docs/resources/waf_dedicated_alarm_masking_rule_v1.md]: wrong resource name

File: docs/resources/waf_dedicated_web_tamper_rule_v1.md [hunk 27]
```diff
@@ -1,9 +1,14 @@
 ---
 subcategory: "Dedicated Web Application Firewall (WAFD)"
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_waf_dedicated_web_tamper_rule_v1"
+sidebar_current: "docs-opentelekomcloud-resource-waf-dedicated-web_tamper_rule-v1"
```
Comment (artem-lifshits) [inline docs/resources/waf_dedicated_web_tamper_rule_v1.md]: underscore (_) and dash (-) mixture in name

**Review by artem-lifshits — CHANGES_REQUESTED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**